etc/vector/vector.d/10-php.toml.tmpl

```diff
@@ -46,3 +46,29 @@ source = '''
 .chan = "error"
 .parsed = parse_regex(.message, r'^(?P<ts>\[\d+\-\w+\-\d+ \d+:\d+:\d+\]) (?P<lvl>\w+): (?P<msg>.*)$') ?? {}
 '''
+
+{{ if getenv "METRICS_ENABLED" "false" }}
+{{ if getenv "METRICS_PHP_FPM_ENABLED" "true" }}
+[sources.php_fpm_metrics_source]
+type = "prometheus_scrape"
+endpoints = [
+  "http://localhost:10001/fpm/dp_default/status?openmetrics",
+  "http://localhost:10001/fpm/dp_broadcaster/status?openmetrics",
+  "http://localhost:10001/fpm/dp_gql/status?openmetrics",
+  "http://localhost:10001/fpm/dp_internal/status?openmetrics",
+]
```

Comment:
Not sure if you are relying on the backlog count to be accurate here but AFAIK, the backlog count always returns 0 (https://bugs.php.net/bug.php?id=80739) - or at least it certainly did up until PHP 8.1.
I've had to use this command in a loop if I actively wanted to monitor the backlog status: `ss -ln | grep "/run/php_fpm_dp" | grep -v status | awk '{print "["$3"]: "$5}'`
```
[0]: /run/php_fpm_dp_default.sock
[0]: /run/php_fpm_dp_internal.sock
[0]: /run/php_fpm_dp_gql.sock
[0]: /run/php_fpm_dp_broadcaster.sock
```

Reply:
wasn't aware of that, thanks

the metrics I have immediate use for are mostly the number of active/idle/total processes (even though it seems like this number may not be always accurate: https://github.com/php/php-src/issues/14212)